### PR TITLE
remove warning banner about views with encryption support by crud v7

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/development_suite/api-console/api-design/mongo_views.md [hunk 1]
```diff
@@ -11,11 +11,6 @@ sidebar_label: Create Views
 
 ## Create a new View
 
-:::danger
```
Comment (fredmaggiowski) [inline docs/development_suite/api-console/api-design/mongo_views.md:14]: If I use a v7 version prior the fix I still can't use the feature, wouldn't it be better to put this as a lower level warning stating the actual version that do not support views over encrypted fields?

File: docs/development_suite/api-console/api-design/mongo_views.md [hunk 2]
```diff
@@ -11,11 +11,6 @@ sidebar_label: Create Views
 
 ## Create a new View
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/mongo_views.md:13]:
```suggestion

:::info
The creation of Views cointaining [encrypted fields](/runtime_suite/crud-service/30_encryption_configuration.md) is supported for [CRUD service](/runtime_suite/crud-service/10_overview_and_usage.md) versions < v7.0.0 and >= v7.1.0.  
Please, pay attention to the CRUD service version when using this feature.
:::
```

File: docs/development_suite/api-console/api-design/mongo_views.md (hunk 1 above)
Comment (danibix95) [reply docs/development_suite/api-console/api-design/mongo_views.md:14]: Is `info` level sufficient?

**Review by danibix95 — APPROVED** [review]

File: docs/development_suite/api-console/api-design/mongo_views.md (hunk 1 above)
Comment (albertotessarotto) [reply docs/development_suite/api-console/api-design/mongo_views.md:14]: In the suggestion I converted it in an info banner by specifying the range of versions the does not support the feature. Do you think that would be okay?

**Pull request merged**